admin/qtx_admin_settings.php

```diff
@@ -693,6 +705,17 @@ private function add_advanced_section() {
                         printf( __( 'The color in use is taken from your profile option %s, the third color.', 'qtranslate' ), '"<a href="' . admin_url( '/profile.php' ) . '">' . qtranxf_translate_wp( 'Admin Color Scheme' ) . '</a>"' ) ?></p>
                 </td>
             </tr>
+            <tr id="option_slugs_enabled">
+                <th scope="row"><?php _e( 'Slugs translation', 'qtranslate' ) ?></th>
+                <td>
+                    <label for="slugs_enabled">
+                        <input type="checkbox" name="slugs_enabled"
+                               id="slugs_enabled"
+                               value="1"<?php checked( $q_config['slugs_enabled'] ) ?>/>&nbsp;<?php _e( 'Enable slugs translation.', 'qtranslate' ) ?>
+                    </label>
+                    <p class="qtranxs-notes"><?php echo __( 'This will activate the slug translation module (experimental feature). ', 'qtranslate' ) ?></p>
+                </td>
+            </tr>
```

Comment:
I think this can move to the integration panel, because it is directly related.
This functionality should certainly exist for any module in the future.